pkg/kotsclient/channel.go

```diff
@@ -160,3 +160,23 @@ func (c *VendorV3Client) ArchiveChannel(appID, channelID string) error {
 
 	return nil
 }
+
+func (c *VendorV3Client) UpdateSemanticVersioning(appID string, channel *channels.AppChannel, enableSemver bool) error {
+	request := types.UpdateChannelRequest{
```

Comment:
from the reference page, it seems like the app id, channel id and name are all required, but i only see the name passed here. https://replicated-vendor-api.readme.io/v3/reference/updatechannel

Reply:
ah. I got lazy and passed the whole channel instead. If it's more clear I'll pass them both separately